server/expression/data_store.go

```diff
@@ -21,10 +25,24 @@ func (ds AttributeDataStore) Source() string {
 	return "attr"
 }
 
+func (ds AttributeDataStore) getFromAlias(name string) (string, error) {
+	alias, found := AttributeAlias[name]
+	if found {
+		value := ds.Span.Attributes.Get(alias)
+		if value == "" {
+			return "", fmt.Errorf(`attribute "%s" not found`, alias)
+		}
+
+		return value, nil
+	}
```

Comment:
I think we can reduce nesting here: 
```suggestion
	if !found  {
	  return "", fmt.Errorf(`attribute "%s" not found`, alias)
	}
	value := ds.Span.Attributes.Get(alias)
	if value == "" {
		return "", fmt.Errorf(`attribute "%s" not found`, alias)
	}
	
	return value, nil
```